src/Command/InfectionCommand.php

```diff
@@ -165,6 +166,13 @@ protected function configure(): void
                 InputOption::VALUE_NONE,
                 'Debug mode. Will not clean up Infection temporary folder.'
             )
+            ->addOption(
+                'error-timeout',
```

Comment:
Do we really need to expose this option to our end users?I think it would lead to confusion, as its not relevant for most users.

Do you think we could chose a sensible default, and simply use that?

Reply:
Honestly, I'm not sure. In this particular instance (as it relates to codeception), it really only is a microsecond difference. My suggestion is leave it as a config file option and remove the cli arg, if you do want to remove it. That way, it isn't a mainstream option, but can still be configured.